experimental/dds/tree2/src/test/shared-tree/sharedTree.bench.ts

```diff
@@ -424,6 +424,45 @@ describe("SharedTree benchmarks", () => {
 			});
 		}
 	});
+
+	// Note that the times reported by this benchmark represent the computation of several peers.
+	// In practice, this computation is distributed across peers, so the actual time reported should be
+	// divided by the number of peers.
+	describe("rebasing commits", () => {
+		const commitCounts = [1, 10, 20];
+		const nbPeers = 5;
+		for (const nbCommits of commitCounts) {
+			benchmark({
+				type: BenchmarkType.Measurement,
+				title: `for ${nbCommits} commits per peer x ${nbPeers} peers`,
```

Comment:
```suggestion
				title: `for ${nbCommits} commits per peer for ${nbPeers} peers`,
```
I read this as "Peer X" not as "peer times..."